EIPS/eip-1238.md

```diff
@@ -0,0 +1,152 @@
+  /// @notice Find the address bound to an ERC1238 soulbound token (short "SBT")
+  /// @dev SBTs assigned to zero address are considered invalid, and queries
+  ///  about them do throw.
+  /// @param _tokenId The identifier for an SBT
+  /// @return The address of the owner bound to the SBT
+  function boundTo(uint256 _tokenId) external view returns (address);
+}
+
+interface ERC165 {
+  /// @notice Query if a contract implements an interface
+  /// @param interfaceID The interface identifier, as specified in ERC-165
+  /// @dev Interface identification is specified in ERC-165. This function
+  ///  uses less than 30,000 gas.
+  /// @return `true` if the contract implements `interfaceID` and
+  ///  `interfaceID` is not 0xffffffff, `false` otherwise
+  function supportsInterface(bytes4 interfaceID) external view returns (bool);
+}
+
+/// @title ERC-721 Non-Fungible Token Standard, optional metadata extension
+/// @dev See https://eips.ethereum.org/EIPS/eip-721
+///  Note: the ERC-165 identifier for this interface is 0x5b5e139f.
+interface ERC721Metadata /* is ERC721 */ {
+  /// @notice A descriptive name for a collection of NFTs in this contract
+  function name() external view returns (string _name);
+
+  /// @notice An abbreviated name for NFTs in this contract
+  function symbol() external view returns (string _symbol);
+
+  /// @notice A distinct Uniform Resource Identifier (URI) for a given asset.
+  /// @dev Throws if `_tokenId` is not a valid NFT. URIs are defined in RFC
+  ///  3986. The URI may point to a JSON file that conforms to the "ERC721
+  ///  Metadata JSON Schema".
+  function tokenURI(uint256 _tokenId) external view returns (string);
+}
+```
+
+For "ERC721 Metadata JSON Schema", see `ERC721` document.
+
+### Caveats
+
+No caveats known for now.
+
+## Rationale
+
+Since 2018, the Ethereum community has been seeking consensus for a standard
+for non-transferrable non-fungible token.
+
+For `ERC1238`, we propose:
+
+- `event Bond(address indexed _from, address indexed _to, uint256 indexed _tokenId)`, essentially `ERC721`'s Transfer event but renamed to "Bond".
+- `function boundTo(uint256 _tokenId) external view returns (address)`,
+  essentially `ERC721`'s `ownerOf(uint256 _tokenId)` but renamed to `boundTo`.
+
+The benefit of using the `ERC721Metadata` interface is that implementers (e.g.
+wallets) of `ERC1238` can identify transfer and tracking functionality using the
+`ERC165` `supportsInterface(bytes4 interfaceID)` function.
+
+## Backwards Compatibility
+
+We have adopted the `ERC165` and `ERC721Metadata` functions. `ERC1238`'s
+`boundTo` function is the pendant to `ERC721`'s `ownerOf` function.
+
+## Test Cases
+
+See "Implementations" section.
+
+## Implementations
+
+- [rugpullindex/ERC1238](https://github.com/rugpullindex/ERC1238) is a minimal
+  implementation using `burn` and `mint` and test cases released under GNU
+  General Public License v3.0.
+
```

Comment:
This is not a valid EIP section.  There is a reference implementation section, but it should include a barebones reference implementation inline or in `../assets/eip-4966/*`.